qtopconf.yaml

```diff
@@ -50,6 +50,13 @@ state_abbreviations:
     E: exiting_of_user
     W: waiting_of_user
     H: holding_of_user
+    B: busy_of_user
+    F: failed_of_user
+    M: moo_of_user
+    S: suspended_of_user
+    T: exiting_of_user
+    U: ufo_of_user
```

Comment:
ok, to be fair, this and the moo above are a bit fishy, but not blocker for now.
the real issue is that the states should either not have long description OR that should be used somewhere, fi. help
(and be accurate!)